src/modules/siptrace/doc/siptrace_admin.xml

```diff
@@ -18,24 +18,17 @@
 	<para>
 		The SIPtrace module offer a possibility to store incoming and outgoing SIP
 		messages in a database and/or duplicate to the capturing server (using <acronym>HEP</acronym>,
-		the Homer encapsulation protocol, or plain SIP mode)
+		the Homer encapsulation protocol, or plain SIP mode). Since version 5.4 new levels of tracing
+		are available. Transactions and dialogs can be traced. Trace flag is now useless.
```

Comment:
"flag is useless" - probably should be changed after the re-introducing as well.

Reply:
thanks, fixed